plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -18,7 +18,7 @@
         <mavenCoordinates
               artifactId="appengine-api-1.0-sdk"
               groupId="com.google.appengine" 
-              version="1.9.59" />
+              version="1.9.60" />
```

Comment:
Maybe we should make our update-to-latest more explicit and support using `version` as `LATEST` or `RELEASE`?

Reply:
Good point.  In these cases the user won't be able to fetch the artifacts either.  I wonder if we wouldn't be better treating this as a fatal error and abort the current operation.

Reply:
Perhaps it is possible that, although fetching a latest version from MC fails, a fallback version exists locally and the user can continue the operation.

Reply:
Not necessarily. It's very common to have many old versions sitting around in your local repo.